packages/common/src/resourceLabel.ts

```diff
@@ -0,0 +1,11 @@
+export type ResourceLabel = `${string}__${string}`;
```

Comment:
if you wanted to get real fancy, could make this generic, e.g.

```ts
export type ResourceLabel<namespace extends string = string, name extends string = string> = `${namespace}__${name}`;
```

Reply:
Ohhh I like that! I was thinking `namespace` and `name` should kinda be types